pkgs/os-specific/linux/sssd/default.nix

```diff
@@ -18,6 +18,8 @@ stdenv.mkDerivation rec {
     url = "https://fedorahosted.org/released/sssd/${pname}-${version}.tar.gz";
     sha256 = "0ngr7cgimyjc6flqkm7psxagp1m4jlzpqkn28pliifbmdg6i5ckb";
   };
+  # https://github.com/SSSD/sssd/pull/976
```

Comment:
Reading the PR, it does not work and is buggy. Please use https://github.com/SSSD/sssd/commit/bc56b10aea999284458dcc293b54cf65288e325d via `fetchpatch` instead.

Reply:
Done. I made change on the top of your PR. Is this right?

Reply:
Wait. I didn't use `fetchpatch`. fixing it.

Reply:
Done.